Source/Device.swift.gyb

```diff
@@ -418,6 +419,11 @@ public enum Device {
       case "i386", "x86_64", "arm64": return simulator(mapToDevice(identifier: ProcessInfo().environment["SIMULATOR_MODEL_IDENTIFIER"] ?? "watchOS"))
       default: return unknown(identifier)
       }
+#elseif os(macOS)
+  switch identifier {
+  case "Mac14,6": return macBookProM2Max
+  default: return unknown(identifier)
+  }
```

Comment:
Please move macOS-related code to a separate PR. Also, please use the gyb-mechanism to generate this code instead of hardcoding it.